feature-libs/qualtrics/components/qualtrics-loader/qualtrics-loader.service.ts

```diff
@@ -50,16 +57,19 @@ export class QualtricsLoaderService implements OnDestroy {
    */
   protected qsi$: Observable<any> = this.qsiLoaded$.pipe(
     switchMap(() => this.isDataLoaded()),
-    map((dataLoaded) =>
-      dataLoaded ? this.winRef?.nativeWindow['QSI'] : EMPTY
-    ),
-    filter((api) => Boolean(api)),
-    tap((qsi) => (this.qsiApi = qsi))
+    map((dataLoaded: boolean) => (dataLoaded ? this.window?.QSI : EMPTY)),
+    filter((api: any) => Boolean(api)),
+    tap((qsi: any) => (this.qsiApi = qsi))
   );
 
+  get window(): QualtricsWindow | undefined {
+    return this.winRef?.nativeWindow;
+  }
+
   constructor(
     protected winRef: WindowRef,
-    protected rendererFactory: RendererFactory2
+    protected rendererFactory: RendererFactory2,
+    @Inject(PLATFORM_ID) protected platformId: any
```

Comment:
A breaking change? 🤔

Reply:
As discussed, it is most likely not due to the fact that it still exist in the storefrontlib, where the feature used to live.

Reply:
Migration plan was to just put deprecation comments in the storefrontlib and move everything to the feature-lib, so I believe it is not a breaking change, but a 'new' feature